doc/mlsys_paper/paper.tex

```diff
@@ -378,18 +380,26 @@ \section{Example: Learning Linear Regression Models}
 types, respectively~\cite{sanderson2016armadillo}.
 
 % Indented a little to make it clear it's different from the text...
+
+%% CS: "accu((y - X * theta) % (y - X * theta))" is redoing computations
+%% CS: ie. efficiency problem
+%% CS: how about:
+%% CS:   arma::mat tmp = y - X * theta;
+%% CS:   return arma::accu(tmp % tmp);  // accumulate element-wise product
+%% CS: in the above code, the comment hints at what the % operator does,
+%% CS: as otherwise it's potentially confusing to someone not familiar with Armadillo
```

Comment:
Agree, the only problem would be that it makes the code a bit longer.  How about:

```
return std::pow(arma::norm(y - X * theta), 2.0);
```

(I didn't like the extra usage of `std::pow()` there though originally.)

Reply:
ok with me.